### ISSUE TITLE
Dont release new versions without release notes and milestones

### ISSUE THREAD

**Issue by MTechDE** [kwhitley/apicache#48]
It is hard to follow the changes, without any milestones or release notes.
Also you should not use "intern" versions and then dont release them and jump directly to for example 0.7.0.

**Comment by kwhitley** [comment]
@MTechDE Point noted (and valid)... however, as I'm using standard [semantic versioning](http://semver.org/), there will often be minor releases in a row without a patch/bugfix release between (as I'm adding features).

I'll certainly spend a bit more effort filling in release notes though, cheers~

👍
